README.asciidoc

```diff
@@ -369,6 +370,29 @@ command names:
 ------------------------------------------------------------------------------------------
 
 
+[[Piping]]
+Hints on using image-bootstrap within a pipe
+--------------------------------------------
+If you want to run *image-bootstrap* in a pipe to capture its output to both
+`stdout/stderr` in a single log file, be sure to run *image-bootstrap* in
+unbuffered mode, e.g.:
+
+----------------------------------------------------
+python -u image-bootstrap [params] 2>&1 | tee my.log
+----------------------------------------------------
+
+The default shebang generated by python setuptools does not use `-u`. It's also
+not easily possible to have it pass `-u` since the `env` command does allow for
+for passing command parameters only in fairly recent versions. As a consequence,
+`stdout/stderr` won't be synchronized and error output you see in a log file
+will not exactly correspond to preceding/succeeding output on `stdout`. (See
+https://github.com/hartwork/image-bootstrap/issues/71 for more details)
+
+Moreover, if you're using `bash` and you need to keep track of
+*image-bootstrap*'s exit code, be sure to run `set -o pipefail` prior to
+invoking *image-bootstrap*. (Please google `bash pipefail` for more details.)
```

Comment:
Let's not advertise Google here. Let's use one of the other options:
* `help set | fgrep -A2 pipefail`
* `man bash`
* something like: `section https://www.gnu.org/software/bash/manual/html_node/Pipelines.html[3.2.2 Pipelines] of the https://www.gnu.org/software/bash/manual/html_node/index.html[GNU Bash Reference Manual]`

Reply:
ack